cli/unstable_progress_bar.ts

```diff
@@ -146,14 +151,11 @@ export class ProgressBar {
   /**
    * Constructs a new instance.
    *
-   * @param writable The {@link WritableStream} that will receive the progress bar reports.
    * @param options The options to configure various settings of the progress bar.
    */
-  constructor(
-    writable: WritableStream<Uint8Array>,
-    options: ProgressBarOptions,
-  ) {
+  constructor(options: ProgressBarOptions) {
     this.#options = {
+      writable: options.writable ??= Deno.stdout.writable,
```

Comment:
Can we make this `stderr` by default based on #5495? I started a thread with @kt3k about changing `spinner` to use `stderr` by default in #6350 as well.

Reply:
I'm not a big fan of there being a default, but if there is one, based off what Andrew is saying here, it makes sense for it to be stderr

Reply:
@BlackAsLight - I'm curious your reasons for not wanting a default here?

Reply:
I just think it's better for people to make the conscious choice about where it's going than perceive it as magic.

Reply:
Done